amz-review-scraper/app.py

```diff
@@ -4,31 +4,42 @@
 from flask_migrate import Migrate
 import sys
 import amzscraper as amazon
+import urls
 
 app = Flask(__name__)
 app.config.from_object(Config)
 db = SQLAlchemy(app)
 migrate = Migrate(app, db)
-# this import must go here
-import models
 
-def main():
-    asin = input("Please enter a vaild ASIN: ")
-    url =  amazon.create_url(asin)
-
-    result = amazon.scrape(url, asin)
-
-    reg = models.Items(name=result.name, customer_reviews_count=result.count, asin=asin)
+# Valid ASINs are a combination of 10 Letters and Integers
+# Not sure yet how to check is a seemingly valid ASIN is actually
+# valid without first trying to scrape it
+# example of seemingly valid ASIN = B011111111
+def get_valid_asin():
```

Comment:
Seems like a great method to isolate and test. Then as you figure out more details of what makes a valid ASIN, you can add more tests.